src/app/ClusterInfo.h

```diff
@@ -0,0 +1,42 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app/AttributePathParams.h>
+#include <app/util/basic-types.h>
+
+namespace chip {
+namespace app {
+struct ClusterInfo
+{
+    ClusterInfo(AttributePathParams & aAttributePathParams, bool aDirty) :
```

Comment:
`const AttributePathParams&` ?

Reply:
AttributePathParams could be changed later.

Reply:
yes, but you are creating a copy here. This should be const.

Reply:
yes, done, thanks